response_operations_ui/views/collection_exercise.py

```diff
@@ -177,9 +179,17 @@ def view_collection_exercise(short_name, period):
         validation_failed=validation_failed,
         show_msg=show_msg,
         info_panel=info_panel,
+        total_ci_count=total_ci_count,
```

Comment:
We have a context builder for the table which would be a better spot for this, you will have to change it from list to dict, you can even include the display logic in there (Not sure about that one as you end up with a double if)

Reply:
{
   "total_ci_count": count,
   "display_header": true false 
   "cis" : [original_list]
}

Reply:
Thinking about it forget the display_header in the dict, we want to eliminate extra logic not add it. Just do a if in the jinga

Reply:
Code has been updated based on suggestions in the comment